domain/src/main/java/com/teamagam/gimelgimel/domain/alerts/UpdateLatestInformedAlertTimeInteractor.java

```diff
@@ -0,0 +1,39 @@
+package com.teamagam.gimelgimel.domain.alerts;
+
+import com.google.auto.factory.AutoFactory;
+import com.google.auto.factory.Provided;
+import com.teamagam.gimelgimel.domain.alerts.entity.Alert;
+import com.teamagam.gimelgimel.domain.alerts.repository.InformedAlertsRepository;
+import com.teamagam.gimelgimel.domain.base.executor.ThreadExecutor;
+import com.teamagam.gimelgimel.domain.base.interactors.BaseDataInteractor;
+import com.teamagam.gimelgimel.domain.base.interactors.DataSubscriptionRequest;
+
+import java.util.Collections;
+
+import rx.Observable;
+
+@AutoFactory
+public class UpdateLatestInformedAlertTimeInteractor extends BaseDataInteractor {
```

Comment:
Basically we're using interactors for everything kind of action we want to make on our domain.
It's a good for several reasons, like uniformly manipulating domain from app and allows for logic to be implemented independently for each "use-case" (=interactors).